terraform/helm/indexer/templates/psp.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
```

Comment:
PSP is going to be deprecated soon if you want to take a look of the replacement https://kubernetes.io/docs/concepts/security/pod-security-policy/

Reply:
Yea I saw that. We likely need to do PSP deprecation everywhere all at once in a separate PR

Reply:
how about we start with not adding more psp :D

Reply:
Then indexer will fail to schedule since PSP is blocking currently in our deploys

Reply:
hmm okay looks like the psp replacement is still in beta on 1.22